packages/utils/testing/src/lib/plugin.js

```diff
@@ -108,7 +108,7 @@ class PluginsAssert {
     this.plugins = plugins;
   }
   actionForEventRegistered(name, action) {
-    assert(this.plugins.plugin.listeners[name] && this.plugins.plugin.listeners[name].some(registered => registered.raw === action), `action for ${name} wanted, but not registered`);
```

Comment:
This condition would always fail. I've simplified it to simply check if a handler was registered for a given event for now. 

This API isn't used anywhere yet (apart from this PR), so it's a save change.